source/fundamentals/databases-collections.txt

```diff
@@ -0,0 +1,185 @@
+=========================
+Databases and Collections
+=========================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB organizes data into a hierachy of the following levels:
+
+- databases
+
+- collections
+
+- documents
+
+Databases are the top level of data organization in a MongoDB instance.
+Databases are organized into collections, which contain documents.
+Documents contain literal data like strings, numbers, and dates as well
```

Comment:
Is this a standard description of documents that we use elsewhere?  It seems incomplete, e.g. it doesn't mention that it contains both keys and values, or arrays. Since the document model is a huge differentiator for us, maybe this should instead link to some other page that describes the document model in more detail.

Reply:
Good point -- I've left the short explanation there for users who just need a refresher/don't want to read the in-depth manual page on documents, but I've added a link to the official manual page explaining documents.